Sources/Impl/PinLayout+Layouting.swift

```diff
@@ -133,7 +133,12 @@ extension PinLayout {
         if !validateComputedHeight(newRect.size.height) {
             newRect.size.height = view.getRect(keepTransform: keepTransform).height
         }
-        
+
+        if Pin.autoSizingInProgress, let autoSizeCalculable = view as? AutoSizeCalculable {
+            let marginInsets = PEdgeInsets(top: -_marginTop, left: -_marginLeft, bottom: -_marginBottom, right: -_marginRight)
+            autoSizeCalculable.setAutoSizingRect(newRect, margins: marginInsets)
+        }
+
```

Comment:
Should we add an `else` here? I don't think we want/need to call `view.setRect(newRect, keepTransform: keepTransform)` below?

Reply:
True, I also deleted the comment above `view.setRect` as it is only relevant for the concrete implementation of `setRect` for `UIView` and the exact same comment is present there.